packages/mdc-typography/README.md

```diff
@@ -2,22 +2,33 @@
 title: "Typography"
 layout: detail
 section: components
-excerpt: "Material Design typography guidelines implemented via CSS."
+excerpt: "Typographic scale that handles a set of type sizes"
 iconId: typography
 path: /catalog/typography/
 -->
 
 # Typography
 
-MDC typography is a CSS-only component that implements the
-[Material Design typography guidelines](https://material.io/guidelines/style/typography.html), and makes them available to
-developers as CSS classes.
+Material Design's typographic scale has a limited set of type sizes that work well together along with the layout grid.
+
+MDC Typography is a foundational module that applies that typographic scale to MDC Web components. The typographic styles in this module are derived from ten styles:
+
+* Display 4
+* Display 3
+* Display 2
+* Display 1
+* Headline
+* Title
+* Subheading
```

Comment:
What happened to Subheading 2 and Subheading 1?  (Also applicable below under CSS styles and $style values; afaict both variables still exist?)